testing/features/support/helpers/page.rb

```diff
@@ -31,6 +31,23 @@ def switch_to_newly_opened_window!(new_page: false)
       wait_for_new_url if firefox? && new_page
     end
 
+    def send_tabs(count)
+      count.times do
+        if safari?
+          send_keys(%i[alt tab])
+          # In Safari the modifier key (OPTION, but called :alt), is not
+          # released after being used in the alt+tab call. This means that
+          # subsequent calls do things like alt+click which downloads a link.
+          #
+          # See: https://bugs.webkit.org/show_bug.cgi?id=219948
```

Comment:
Can we keep the date tracker on this (It's a style myself / Quinn trying to keep to) So LH - Dec 2020

Reply:
oops sorry - a copy pasta error :-)